.eslintrc.json

```diff
@@ -14,6 +14,7 @@
     "__static": true
   },
   "rules": {
+    "linebreak-style":"off",
```

Comment:
The `--fix` option for eslint should allow us to not need this rule.

Reply:
woops 🙈 I always enable this because I get this issue in windows in every file/line with the difference of linestyle :P forgot to turn it off